core/arch/arm/kernel/boot.c

```diff
@@ -578,6 +578,11 @@ void *get_dt(void)
 	return fdt;
 }
 
+bool is_embedded_dt(void *fdt)
+{
+	return (fdt && fdt == get_embedded_dt());
```

Comment:
Can remove parentheses.
I would rather define this function as an static inline function in _boot.h_.

Reply:
Done.